fr-French/strings.xml

```diff
@@ -1,301 +1,301 @@
 <resources>
     <!-- Action -->
-    <string name="activate">activate</string>
-    <string name="activated">activated</string>
-    <string name="deactivate">deactivate</string>
-    <string name="deactivated">deactivated</string>
-    <string name="cancel">Cancel</string>
-    <string name="close">Close</string>
-    <string name="confirm">Confirm</string>
-    <string name="copy">Copy</string>
-    <string name="copied">Copied</string>
-    <string name="create">Create</string>
-    <string name="created">Created</string>
-    <string name="delete">Delete</string>
-    <string name="done">Done</string>
-    <string name="modify">Modify</string>
-    <string name="retry">Retry</string>
-    <string name="save">Save</string>
+    <string name="activate">activer</string>
+    <string name="activated">activé</string>
+    <string name="deactivate">désactiver</string>
+    <string name="deactivated">désactivé</string>
+    <string name="cancel">Annuler</string>
+    <string name="close">Fermer</string>
+    <string name="confirm">Confirmer</string>
+    <string name="copy">Copier</string>
+    <string name="copied">Copié</string>
+    <string name="create">Créer</string>
+    <string name="created">Créé</string>
+    <string name="delete">Supprimer</string>
+    <string name="done">Terminé</string>
+    <string name="modify">Modifier</string>
+    <string name="retry">Réessayer</string>
+    <string name="save">Sauvegarder</string>
     <!-- Left menu -->
     <string name="alias">alias</string>
-    <string name="aliases">aliases</string>
-    <string name="mailbox">mailbox</string>
-    <string name="mailboxes">mailboxes</string>
-    <string name="settings">Settings</string>
-    <string name="about">About</string>
-    <string name="rate_us">Rate Us</string>
-    <string name="sign_out">Sign Out</string>
-    <string name="sign_out_confirmation">You will be signed out</string>
-    <string name="yes_sign_me_out">Yes, sign me out</string>
+    <string name="aliases">alias</string>
+    <string name="mailbox">boîte mails</string>
+    <string name="mailboxes">boîtes mails</string>
```

Comment:
`Boîte mail` or `Boîte mails` ? I'm not sure which is better and above all what's right...